packages/web-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -90,7 +97,12 @@ const Tooltip = (props: SpiritTooltipProps) => {
         y,
       }}
     >
-      <div ref={tooltipRef} {...styleProps} {...otherProps}>
+      <div
+        ref={tooltipRef}
+        className={classNames(classProps.rootClassName, styleProps.className)}
+        style={styleProps.style}
+        {...otherProps}
```

Comment:
Pleas move `otherProps` to the starting position, so it will not be possible to override the existing props.

Reply:
moved 👍